### ISSUE TITLE
Duplicate data?

### ISSUE THREAD

**Issue by namnatulco** [UlmApi/kleinerspatz#13]
It looks like there is erroneous or duplicate data in the current JSON file; see "St. Barbara-Straße 35" on the map (near the Donau, to the south along the Illerstraße). It may indicate that our parsing script doesn't work as intended (or we merged some things incorrectly).

**Comment by hkopp** [comment]
Yes, you are right. But the bug is not in the parser, but in the initial data.

In our json-file there is
{"telefon":"0731 161 51 44 ","name":"St. Barbara-Stra&szlig;e 35 (Stadt Ulm) ","over3":"false","mail":"st-barbara-strasse@ulm.de","wgs84-north":48.38602614,"id":"69","wgs84-east":9.97662014}
and
{"telefon":"31127","name":"St. Barbara-Stra&szlig;e 35 (Montessori)","over3":"true","mail":"strolche@montessori-ulm.de","wgs84-north":48.38602614,"id":"89","wgs84-east":9.97662014}
which seem to be different, despite having the same coordinates.

**Comment by berb** [comment]
Same address, but different Kitas in the same building: http://www.ulm.de/leben_in_ulm/bildungshaus_ulmer_spatz_ulm_weststadt.83159.3076,3665.htm

**Comment by namnatulco** [comment]
Yes, but that means you can only click on one of them :-).
